topics/climate/tutorials/cmip6/tutorial.md

```diff
@@ -84,34 +84,23 @@ CMIP6 is the Coupled Model Intercomparison Project in its 6th phase. [CMIP6](htt
 * Calculate an ensemble mean of all used models
 * Calculate and plot the seasonal mean 
   
-## Get data
+## Starting Galaxy Climate JupyterLab
 
-> ### {% icon hands_on %} Hands-on: Data upload
+ ## {% icon hands_on %} Hands-on: Launch JupyterLab for Ocean / Atmosphere / Land / Climate Python ecosystem in Galaxy
 >
-> 1. Create a new history for this tutorial
-> 2. Import the files from [Zenodo]() or from the shared data library
->
->    ```
->    
->    ```
->    ***TODO***: *Add the files by the ones on Zenodo here (if not added)*
->
->    ***TODO***: *Remove the useless files (if added)*
->
->    {% snippet faqs/galaxy/datasets_import_via_link.md %}
->    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
->
-> 3. Rename the datasets
-> 4. Check that the datatype
->
->    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="netcdf" %}
->
-> 5. Add to each database a tag corresponding to ...
->
->    {% snippet faqs/galaxy/datasets_add_tag.md %}
+> Currently JupyterLab for Ocean / Atmosphere / Land / Climate Python ecosystem in Galaxy is available on [Live.useGalaxy.eu](https://live.usegalaxy.eu) only. JupyterLab for Ocean / Atmosphere / Land / Climate Python ecosystem and not the default JupyterLab in Galaxy contains all the python packages and additional software we need for running Earth System Model, including Functionally Assembled Terrestrial Ecosystem Simulator (FATES). The default JupyterLab in Galaxy would not be sufficient for executing all the tasks in th
+is tutorial.
 >
+> 1. Open the {% tool [JupyterLab](interactive_tool_jupyter_notebook) %} by clicking [here](https://live.usegalaxy.eu/?tool_id=interactive_tool_climate_notebook){:target="_blank"}
+
+
+> 2. Create a new history for this tutorial
+
 {: .hands_on}
 
+# Exploring the Pangeo CMIP6 catalog
+
+
```

Comment:
Here we need to write down what packages we need to use and how to explore data. Something like:

To explore Pangeo CMIP6 climate data, we are using [intake-esm](https://intake-esm.readthedocs.io/), a data cataloging utility built on top of [intake](https://github.com/intake/intake), [pandas](https://pandas.pydata.org/), and [xarray](https://xarray.pydata.org/en/stable/). 

We start by opening an ESM (Earth System Model) collection definition file e.g. a JSON file that conforms to the ESM Collection Specification. When provided a link/path to an esm collection file, intake-esm establishes a link to a database (CSV file) that contains data assets locations and associated metadata (i.e., which experiment, model, the come from). The collection JSON file can be stored on a local filesystem or can be hosted on a remote server:

```
import intake

col_url = "https://storage.googleapis.com/cmip6/pangeo-cmip6.json"

col = intake.open_esm_datastore(col_url)
```

We can then search and discovery by execurting queries against the catalog:

```
col_subset = col.search(
           experiment_id=["historical", "ssp585"],
           table_id="Oyr",
           variable_id="o2",
           grid_label="gn",
         )
```

Then you can move the first hands-on from **Plot the snowfall for January between 1985 - 2014 from NorESM2-MM** (Search snowfall `prsn` for historical  experiments and  member "r1i1p1f1'). This will be the hands-on for searching and exploring data.

- We can ask to provide metadata information such as units, long name and give the list of available models.

Then we can have the next section on Plotting.